src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { ButtonProps } from '../Button';
+import style from './IconButton.st.css';
+import { ButtonNext } from 'wix-ui-core/button-next';
+
+export interface IconButtonProps extends ButtonProps {
+  icon: React.ReactNode;
+}
+
+interface DefaultProps {}
+
+interface State {}
+
+/** An implementation of an IconButton for TPAs, The TPA implementation provides few extendable theme properties with the ability to override the base properties as well,By default the IconButton will pull the theme from the site (site colors), */
+export class IconButton extends React.Component<IconButtonProps, State> {
+  static displayName = 'IconButton';
+  static defaultProps: DefaultProps = {};
+
+  render() {
+    const { icon, disabled, ...rest } = this.props;
+    return (
+      <ButtonNext
+        aria-label={'icon button'}
```

Comment:
This should be passed by the developer using the component, since the content of the `aria-label` should be translated into the language of the site.
So it's better to leave it to them

Reply:
but it is a default value (if no passed from a user) it'll be overwritten

Reply:
Yeah, I hear what your'e saying.
But still, I think it might be redundant, and maybe not the right product for everyone.
I prefer that it won't say anything if someone leaves this by mistake.
You could say it's kind of removing the responsibility from our side 😄

Reply:
Can't make a11y tests pass without it ```Running a11y test...
1. Testing component IconButton render...
  IconButton render - button: (Impact: critical)
  Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element's default semantics were not overridden with role="presentation"
  Element's default semantics were not overridden with role="none"
  Element has no title attribute or the title attribute is empty
```
may be you could advice me on how to deal with it?

Reply:
I understand...
So for this, inside the `IconButton.meta.tsx` file you can pass an `aria-label` prop with `'share'` for instance

Reply:
So you can remove this now 😄